config/sunspot.yml

```diff
@@ -1,24 +1,23 @@
----
 production:
   solr:
-    hostname: pad.publiclab.org
+    hostname: localhost
     port: 8983
-    read_timeout: 120
     log_level: WARNING
-    path: "/solr/default"
-  disabled: true
```

Comment:
@david-days You need to keep this line. This is the reason the build is failing in Travis CI.